src/moonlink/src/storage/iceberg/rest_catalog_test_utils.rs

```diff
@@ -25,6 +28,29 @@ pub(crate) fn default_rest_catalog_config() -> RestCatalogConfig {
     }
 }
 
+/// Util function to transaction vec into set
+pub(crate) fn to_set<T: Eq + Hash>(vec: Vec<T>) -> HashSet<T> {
```

Comment:
not sure how useful it is, if you really want it, better to put it into a more centralized place so others could also use it.

Reply:
Comment unaddressed?